### PR TITLE
[enhancement](delete) add column id to delete push task

### PR DESCRIPTION
**Pull request by Hastyshell**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/task/PushTask.java [hunk 1]
```diff
@@ -127,33 +130,36 @@ public TPushReq toThrift() {
...
                 for (Predicate condition : conditions) {
                     TCondition tCondition = new TCondition();
                     ArrayList<String> conditionValues = new ArrayList<String>();
+                    SlotRef slotRef = (SlotRef) condition.getChild(0);
+                    String columnName = slotRef.getColumnName();
+                    tCondition.setColumnName(columnName);
+                    int uniqueId = colNameToColDesc.get(columnName).getColUniqueId();
+                    if (uniqueId >= 0) {
+                        tCondition.setColumnUniqueId(uniqueId);
+                    }
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/task/PushTask.java:144]: please also modify code in BE and check the column unique id exists in tablet schema and column name is equal to the column name in delete predicate.
The code maybe in push handler.cpp

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**